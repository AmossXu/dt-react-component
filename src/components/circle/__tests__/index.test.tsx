
import React from 'react';
import { shallow, mount } from 'enzyme';
import renderer from 'react-test-renderer';
import { Circle } from '../index';
describe('test Circle suite', () => {
    it('renders correctly', () => {
        const tree = renderer
            .create(<Circle />)
            .toJSON();
        expect(tree).toMatchSnapshot();
    });
    it('should be selectable by class "circle_default"', function () {
        expect(shallow(<Circle />).is('.circle_default')).toBe(true);
    })
    it('should mount in a full DOM', function () {
        expect(mount(<Circle />).find('.circle_default').length).toBe(1);
    });
})
