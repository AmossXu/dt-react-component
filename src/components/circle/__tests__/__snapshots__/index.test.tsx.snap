// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test Circle suite renders correctly 1`] = `
<div
  className="circle_default"
>
  
</div>
`;
